nixos/modules/services/databases/redis.nix

```diff
@@ -72,7 +72,27 @@ in {
               defaultText = literalExpression ''
                 if name == "" then "redis" else "redis-''${name}"
               '';
-              description = "The username and groupname for redis-server.";
+              description = ''
+                User account under which this instance of redis-server runs.
+
+                ::: {.note}
+                If left as the default value this user will automatically be
+                created on system activation, otherwise you are responsible for
+                ensuring the user exists before the redis service starts.
+              '';
+            };
+
+            group = mkOption {
+              type = types.str;
+              default = config.user;
```

Comment:
`default = redisName name;` might be preferable - but that would change the group for existing configurations that set `user`.